### DISCUSSION TITLE
Should homebrew package have extra features?

### DISCUSSION THREAD

**Discussion by TWSiO** [nushell/nushell#11489] [General]
The current nushell Homebrew package isn't built with the `extra` feature. It currently is built with the `dataframe` feature though. Should the `extra` feature be added as well, and should any other current features be added like `sqlite`?

It used to be a part of the Homebrew package when it was added in Homebrew/homebrew-core#76072 and it was mentioned in #3362 that it should be included, but was removed when `extra` briefly didn't add anything in Homebrew/homebrew-core#130393. `extra` then was changed to have some stuff that's not in default again in #9327.

I presume the preference is still the same as #3362, but I thought I would just check, as well as see if any other features should be included.

**Comment by fdncred** [comment]
yes, if homebrew wants all the commands, it should include --features=dataframe,extra. That should get everything. That's how I build every day.

**Comment by TWSiO** [comment]
Made a pull request for Homebrew and `extra` is now included in the Homebrew package (Homebrew/homebrew-core#159345).
